File: src/main/java/ordilov/randomplay/common/interfaces/GoogleRefreshToken.java
package ordilov.randomplay.common.interfaces;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;

@Getter
public class GoogleRefreshToken {

  private String scope;
  @JsonProperty("id_token")
  private String idToken;
  @JsonProperty("token_type")
  private String tokenType;
  @JsonProperty("expires_in")
  private String expiresIn;
  @JsonProperty("access_token")
  private String accessToken;
}

File: src/main/java/ordilov/lissn/common/adapter/GoogleRefreshToken.kt
package ordilov.lissn.common.adapter

import com.fasterxml.jackson.annotation.JsonProperty

data class GoogleRefreshToken(
    val scope: String,
    @JsonProperty("id_token") val idToken: String,
    @JsonProperty("token_type") val tokenType: String,
    @JsonProperty("expires_in") val expiresIn: String,
    @JsonProperty("access_token") val accessToken: String
)